core/templates/pages/exploration-player-page/services/content-translation-manager.service.ts

```diff
@@ -121,22 +128,27 @@ export class ContentTranslationManagerService {
       languageCode: string,
       content: SubtitledHtml|SubtitledUnicode
   ): void {
-    const writtenTranslation = writtenTranslations.translationsMapping[
-      content.contentId][languageCode];
+    const writtenTranslation: WrittenTranslation | null = (
+      content.contentId ? writtenTranslations.translationsMapping[
+        content.contentId][languageCode] : null);
```

Comment:
```suggestion
    const writtenTranslation: WrittenTranslation | null = (
      content.contentId ? 
      writtenTranslations.translationsMapping[content.contentId][languageCode] : 
      null
    );
```

or maybe even expand this.